code/modules/mob/living/status_procs.dm

```diff
@@ -411,7 +411,7 @@
 /mob/living/SetStuttering(amount, force = 0)
 	//From mob/living/apply_effect: "Stuttering is often associated with Stun"
 	if(status_flags & CANSTUN || force)
-		stuttering = max(stuttering, 0)
+		stuttering = max(amount, stuttering)
```

Comment:
Oh! My bad! Though this means that one is unable to actually decrease stuttering using this proc. What you want instead:
`stuttering = max(amount, 0)`